Tools/dollymix/ztest.py

```
@@ -0,0 +1,36 @@
+from PIL import Image
+import os, sys
+
+path = os.getcwd()
+basefile = "dollymix"
+ext = ".png"
+size = int(input("size: "))
+offset = float(input("offset: "))
+repeat = int(input("repeat: "))
+
+base = Image.open(os.path.join(path,basefile+ext))
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)